spec/requests/admin/podcasts_spec.rb

```diff
@@ -60,12 +60,13 @@
   end
 
   describe "Updating" do
-    it "updates" do
+    it "updates the podcast" do
       put admin_podcast_path(podcast), params: { podcast: { title: "hello",
-                                                            feed_url: "https://pod.example.com/rss.rss" } }
-      podcast.reload
-      expect(podcast.title).to eq("hello")
-      expect(podcast.feed_url).to eq("https://pod.example.com/rss.rss")
+                                                            feed_url: "https://pod.example.com/rss.rss",
+                                                            published: true } }
+      expect(podcast.reload.title).to eq("hello")
```

Comment:
Just curious can we call `podcast.reload` once here like it was before, or is it necessary to call it several times?

Reply:
@jacobherrington we can call it once! I like how it looks this way, but I'm happy to revert the change, as I have no strong preference. 😊

Reply:
Changed @jacobherrington !

Reply:
Fewer calls is probably faster, but probably not **that** much faster.